site-configs/secondary.ts

```diff
@@ -1,10 +1,26 @@
-import { Config } from "../site-configs";
+import {Environment, GetSiteConfig} from "../site-configs";
```

Comment:
```suggestion
import { Environment, GetSiteConfig } from "../site-configs";
```
seems like prettier (and everyting related eslint/pipeline etc) isn't working